src/Xamarin.Android.NUnitLite/Gui/Instrumentations/TestSuiteInstrumentation.cs

```diff
@@ -103,47 +106,11 @@ public override void OnStart ()
 
 		string GetResultsPath ()
 		{
-			Java.IO.File resultsPathFile = GetExternalFilesDir ();
```

Comment:
PR message should mention removal of this code path.

Additionally, since this assembly can be used on targets < API-30, do we *really* want to remove all this code?  Or instead have a runtime platform check for API-30?